src/studio/src/designer/Dockerfile

```diff
@@ -49,21 +49,21 @@ COPY --from=generate-studio-frontend /dist/app-development/app-development.css .
 COPY --from=generate-studio-frontend /dist/dashboard/dashboard.js .
 COPY --from=generate-studio-frontend /dist/dashboard/dashboard.css .
 
-FROM node:14.16.0-alpine3.13 AS generate-designer-js
+FROM node:14.16.1-alpine3.13 AS generate-designer-js
 COPY src/designer/backend/package.json .
 COPY src/designer/backend/package-lock.json .
 RUN npm ci
 COPY src/designer/backend .
 RUN npm run gulp build
 
-FROM mcr.microsoft.com/dotnet/sdk:5.0.202-alpine3.12 AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0.202-alpine3.13 AS build
```

Comment:
We have had some issues with designer running on alpine 3.13 w.r.t. LibGit2Sharp, so this should be tested extensively locally if we are to include this change

Reply:
https://github.com/Altinn/altinn-studio/issues/5992

Looks like this might cause problems, yeah. Maybe we should add a comment about this in the dockerfile? 👍

Reply:
I'll roll back that one and make a comment in the file.

Reply:
What about aspnet in the same file: `mcr.microsoft.com/dotnet/aspnet:5.0.5-alpine3.13 ` - roll back that as well?